src/ansi.rs

```diff
@@ -411,7 +456,7 @@ impl NamedColor {
 pub enum Color {
     Named(NamedColor),
     Spec(Rgb),
-    Indexed(u8),
+    Indexed(usize),
```

Comment:
This needs to be reverted. If you need more space, making it a `u16` would be fine, but `usize` unnecessarily bloats `Color` which has all sorts of unpleasant consequences.